### PR TITLE
bedops: add page

### PR DESCRIPTION
**Pull request by alexpreynolds**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/bedops.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# bedops
+
+> BEDOPS: A fast, highly scalable, and easily-parallelizable genome analysis toolkit.
```
Comment (agnivade) [inline pages/common/bedops.md]: The user already knows the name of the command. So I think we can remove "BEDOPS:".

File: pages/common/bedops.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> Apply efficient set operations on any number of sorted BED or Starch files such as element-of, intersection, merge, difference, partition, etc. and their logical inverses.
+
+- Take the multiset union of two or more BED or Starch files and write the sorted result to {{path/to/output_file}}:
```
Comment (agnivade) [inline pages/common/bedops.md]: We can just mention "output_file" like this. The full path is more suited in the actual example.

File: pages/common/bedops.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`bedops --merge {{path/to/file_1}} [{{path/to/file_2}} {{path/to/file_3}} ...] > {{path/to/output_file}}`
+
+- Partition overlapping intervals in one or more files into disjoint intervals and write the sorted result to {{path/to/output_file}}:
+
+`bedops --partition {{path/to/file_1}} [{{path/to/file_2}} {{path/to/file_3}} ...] > {{path/to/output_file}}`
+
+- For all operations, efficiently apply set operations on specified {{chromosome}} and ignore other chromosomes:
+
+`bedops --chrom {{chromosome}} --operation {{files...}} > {{path/to/output_file}}`
```
Comment (agnivade) [inline pages/common/bedops.md]: Is `--chrom` correct ? Seems like a typo.

File: pages/common/bedops.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Merge the overlapping intervals in one or more files and write the sorted result to {{path/to/output_file}}:
+
+`bedops --merge {{path/to/file_1}} [{{path/to/file_2}} {{path/to/file_3}} ...] > {{path/to/output_file}}`
+
+- Partition overlapping intervals in one or more files into disjoint intervals and write the sorted result to {{path/to/output_file}}:
+
+`bedops --partition {{path/to/file_1}} [{{path/to/file_2}} {{path/to/file_3}} ...] > {{path/to/output_file}}`
+
+- For all operations, efficiently apply set operations on specified {{chromosome}} and ignore other chromosomes:
```
Comment (agnivade) [inline pages/common/bedops.md]: I think the {{}} can be removed from the descriptions. They are generally used in the examples. I suggest ".. efficiently apply set operations on a specified chromosome and ignore .. "

File: pages/common/bedops.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# bedops
+
+> BEDOPS: A fast, highly scalable, and easily-parallelizable genome analysis toolkit.
+> Apply efficient set operations on any number of sorted BED or Starch files such as element-of, intersection, merge, difference, partition, etc. and their logical inverses.
+
+- Take the multiset union of two or more BED or Starch files and write the sorted result to {{path/to/output_file}}:
+
+`bedops --everything {{path/to/file_1}} {{path/to/file_2}} [{{path/to/file_3}} ...] > {{path/to/output_file}}`
```
Comment (agnivade) [inline pages/common/bedops.md]: I get what you are trying to do with the [ ...] notation. In this project, we lean more towards giving concrete examples and letting the user build up complicated examples from them. From what I see, all commands take multiple input files. So, what we can do is - keep this example to have just 2 files, and then have the next example to have 2 query files.

In this way, the user can figure out that they can add more files on command line. And we also avoid the [ ...] notation, which takes a second to figure out.

What do you think ?

File: pages/common/bedops.md (hunk 1 above)
Comment (alexpreynolds) [reply pages/common/bedops.md]: BEDOPS is the name of the toolkit. The `bedops` binary is one tool in this kit. There are other binaries that are part of BEDOPS.

File: pages/common/bedops.md (hunk 2 above)
Comment (alexpreynolds) [reply pages/common/bedops.md]: That's fine.

File: pages/common/bedops.md (hunk 5 above)
Comment (alexpreynolds) [reply pages/common/bedops.md]: I think having some way to denote that three or more files are also possible is useful. Perhaps we could have two examples, one with two files, and another with four or five files, to make it clear that multiple files are allowed.

File: pages/common/bedops.md (hunk 4 above)
Comment (alexpreynolds) [reply pages/common/bedops.md]: I think I just reviewed how other binaries were explained in their own Markdown files, and I saw this convention used there. I'm happy to do things in whichever way you would prefer.

File: pages/common/bedops.md (hunk 3 above)
Comment (alexpreynolds) [reply pages/common/bedops.md]: No, `--chrom` is correct. This is how the user specifies that she is working with a specific chromosome in the input files, and no other chromosomes.

File: pages/common/bedops.md (hunk 4 above)
Comment (agnivade) [reply pages/common/bedops.md]: If this convention is used in other files, that is incorrect and needs to be removed. Sometimes, these things skip reviews and get committed to the repo.

File: pages/common/bedops.md (hunk 5 above)
Comment (agnivade) [reply pages/common/bedops.md]: Sure, I would think three files would be sufficient. We wouldn't want to make the example too long. But up to you.

File: pages/common/bedops.md (hunk 1 above)
Comment (agnivade) [reply pages/common/bedops.md]: Ah I see. Ok we can keep it in that case.

**Comment by alexpreynolds** [comment]
I pushed a commit to this branch that reflects the changes you requested in your comments. Please let me know if there are any remaining issues.

**Comment by agnivade** [comment]
We need to remove the `[ .. ]` convention used in the examples. And just use {{/path/to/file1}} {{/path/to/file2}}

**Comment by sbrl** [comment]
Ping @alexpreynolds

**Comment by alexpreynolds** [comment]
Is there a summary of what changes are needed? I'm trying to use existing tldr documents as a template and I'm not sure what's left to change. Thanks!

**Comment by agnivade** [comment]
Please look at my [comment](https://github.com/tldr-pages/tldr/pull/1516#issuecomment-340139092) right above :point_up:

**Pull request closed without merging**